lib/rescue_unique_constraint.rb

```diff
@@ -39,5 +43,11 @@ def rescue_unique_constraint(index:, field:)
       end
       index_rescue_handler.add_index(index, field)
     end
+
+    def rails_6?
+      Rails.gem_version.segments.first > RAILS_6_VERSION
```

Comment:
Should this be `>= RAILS_6_VERSION` ? `6 > 6` is going to return false?

Reply:
@mcostanza makes a very solid point

Reply:
yep.. test stubbing failed me. Thanks Mike